v2f/src/main/scala/org/broadinstitute/monster/etl/v2f/V2FUtils.scala

```diff
@@ -1,5 +1,6 @@
 package org.broadinstitute.monster.etl.v2f
 
+import upack._
```

Comment:
Nitpick for (maybe) future reference: we usually follow an import-grouping pattern of:
```
java.* imports
<new line>
other imports
<new line>
scala.* imports
```
And then within each block, we sort things alphabetically. This can help when scanning big import lists.

I'm pretty sure our formatting tool can do this for us automatically now. I'll investigate and report back here.

Reply:
Tooling doesn't seem to Just Work yet. Can you run "Optimize Imports" in IntelliJ?